src/api/configureApi.ts

```diff
@@ -46,13 +46,24 @@ type BypassPath = {
   method: string;
 };
 
-const BYPASS_PATHS = [
-  `/favicon.ico`,
-  `${BASE_REQUEST_PATH}/`,
-  `${BASE_REQUEST_PATH}/metrics`,
-  `${BASE_REQUEST_PATH}/health`,
-  `${BASE_REQUEST_PATH}/keycloak-config`,
-].map((path) => ({ url: path, method: 'GET' }));
```

Comment:
I thought this was harder to read and makes it tricky when you want to add a `POST` method